configs/workload/bert.yaml

```diff
@@ -9,7 +9,7 @@ workflow:
  checkpoint: True
  
 dataset: 
- data_folder: ./data/bert/
+ data_folder: data/bert
```

Comment:
Can we by default infer the relative path. I see other files have similar chnages.